src/main/java/net/projectzombie/region_rotation/file/FileBufferController.java

```diff
@@ -0,0 +1,59 @@
+package net.projectzombie.region_rotation.file;
+
+import org.bukkit.World;
+import org.bukkit.plugin.java.JavaPlugin;
+
+import java.util.HashMap;
+
+/**
+ * Used keep track of the FileBuffers and fetch them.
+ * @author Gephery
+ */
+public class FileBufferController
+{
+    private static FileBufferController FILE_CONTROLLER_SINGLETON;
+    private static JavaPlugin PLUGIN;
+
+    private final HashMap<String, FileBuffer> fileBuffers;
```

Comment:
I changed the buildID to make more sense too. It goes by fileName and world or just fileName for central files. Also the buildID is no longer dependant on FilePath.